db/schema.rb

```diff
@@ -106,7 +106,7 @@
     t.datetime "updated_at"
   end
 
-  create_table "users", force: true do |t|
+  create_table "users", force: :cascade do |t|
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.